Backend/db/modify_tables.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pyodbc
+import argparse
+from dotenv import dotenv_values
+
+# Drop tables action currently disabled - want to avoid dropping tables accidentally. You can
+# enable it by removing the commented out parts if you want to.
+
+# Command line arguments handling
+parser = argparse.ArgumentParser()
+group = parser.add_mutually_exclusive_group()
+group.add_argument(
+    "-c",
+    "--create",
+    help="Creates the database tables specified in create-tables.sql with the triggers in create-triggers.sql.",
+    action="store_true"
+)
+group.add_argument(
+    "-p",
+    "--populate",
+    help="Populates the database tables with the test data specified in create_test_data.sql.",
+    action="store_true"
+)
+"""
+group.add_argument(
+    "-d",
+    "--drop",
+    help="Drops the database tables using the commands in drop-tables.sql.",
+    action="store_true"
+)
+"""
+args = parser.parse_args()
+
+BASEDIR = os.path.abspath(os.path.dirname(".."))
+config = dotenv_values((os.path.join(BASEDIR, ".env")))
+print(BASEDIR)
+
+server = config["DB_SERVER"]
+database = config["DB_NAME"]
+username = config["DB_USERNAME"]
+password = config["DB_PASSWORD"]
+
+conn = pyodbc.connect(
```

Comment:
use db/connection.py to create the connection object

Reply:
From db.connection import setup_db_conn gives me a ModuleNotFoundError: No module named 'db', do you know a fix for this?

Reply:
modify_tables.py is in the db directory, so you should only need  `from connection import setup_db_conn`

Reply:
My issue then is that connection can't be resolved as an import, Pylance complaining about missing imports. Seems like there's some path issues with importing, as I had to modify my .vscode/settings.json to fix this same issue for app.py, but it still seems to have an issue with this.